apps/web/components/ui/tambo/edit-with-tambo-button.tsx

```diff
@@ -133,20 +141,39 @@ export function EditWithTamboButton({
 
     setShouldCloseOnComplete(true);
 
-    await sendThreadMessage(prompt.trim(), {
-      streamResponse: true,
-      additionalContext: {
-        inlineEdit: {
-          componentId: component.interactableId,
-          instruction:
-            "The user wants to edit this specific component inline. Please update the component's props to fulfill the user's request.",
-        },
-      },
-    });
+    // Add the component as a context attachment for inline editing (only if valid)
+    const interactableId = component?.interactableId;
+    const componentName = component?.componentName;
+    if (interactableId && componentName) {
+      addContextAttachment({
+        name: componentName,
+        metadata: {
+          componentId: interactableId,
+        } satisfies InteractableComponentMetadata,
+      });
+    }
 
-    // Clear the prompt after successful send
-    setPrompt("");
-  }, [prompt, isGenerating, component, sendThreadMessage]);
+    try {
+      await sendThreadMessage(prompt.trim(), {
```

Comment:
I hope that if contextKey is left out here, it will pick it up from the context, so it shouldn't need to be passed?